x-pack/plugins/canvas/server/feature.test.ts

```diff
@@ -89,6 +91,8 @@ it(`Calls on Reporting whether to include Generate PDF as a sub-feature`, () =>
   mockReportingPlugin = {
     usesUiCapabilities: () => true,
     registerExportTypes: () => {},
+    getSpaceId: jest.fn(),
+    getScreenshots: jest.fn(),
   };
```

Comment:
```suggestion

In the "Calls on Reporting whether to include Generate PDF as a sub-feature" test, we augment the local mock with:
```
  mockReportingPlugin.usesUiCapabilities = () => true;
```
```